lib/clients/helm.js

```diff
@@ -465,6 +454,40 @@ class Helm {
     await this._cmd(`upgrade --install ${name} -f ${target} --version ${this.mainConfig.chartVersions[name]} --namespace ${namespace} ${repo}/${name}`);
   }
 
+  async _installPolkadot(index) {
+    const chartName = 'w3f/polkadot';
+    const chartVersion = this.mainConfig.chartVersions['polkadot'];
+    const name = this._nodeName(index);
+    const values = this._generateValuesFile(index);
+    const relativeValuesFilePath = this.files.relativeValuesFilePath(this.config.name, index);
+    const fullValuesFilePath = this.files.absoluteValuesFilePath(this.config.name, index);
+
+    this.files.writeYAML(fullValuesFilePath, values);
+
+    try {
+      await this._cmd(`upgrade --install ${name} -f ${relativeValuesFilePath} --version ${chartVersion} --namespace ${this.namespace} ${chartName}`);
+    } catch(e) {
+      console.log(`Could not install ${name}: ${e.message}`);
+    }
+
+  }
+
+  async _installSubstrateAlertrules(index) {
```

Comment:
Given that substrate-alertrules doesn't need any special setup, why not using `_installGenericChart` instead?

Reply:
Sorry, i didn't notice that you are leveraging `_generateValuesFile`. The values file is in fact very simple and doesn't need all the settings created by `_generateValuesFile`, maybe we could use `_installTemplatedChart`, wdyt?

Reply:
- To make it similar as it was configured before (embedded alertrules in polkadot chart https://github.com/w3f/polkadot-charts/blob/master/charts/polkadot/templates/alertrules.yaml#L20 ), I need to retrieve this dynamic values: https://github.com/w3f/polkadot-deployer/blob/polka-chart-version/lib/clients/helm.js#L180

- It will be set here: https://github.com/w3f/substrate-alertrules-chart/blob/init/charts/substrate-alertrules/values.yaml#L1

- I could you use the _installTemplatedChart to inject dynamically that value instead of the generic

wdyt ?

Reply:
Yes, that sounds good, `_installTemplatedChart` is a good fit for that use case :+1: